### PR TITLE
[G2M] Remove imports of CSS from carousel

### PR DESCRIPTION
**Pull request by SamSverko**
- Carousel - Remove imports of CSS.
- Convert carousel.css to MUI makeStyles.
- Move react-slick CSS imports to app-level.
- README - Update with required CSS import.

### CODE REVIEW

**Comment by SamSverko** [comment]
**Bonus:** This allows the project to work with Next.js.

**Review by woozyking — APPROVED** [review]
this addresses https://github.com/EQWorks/react-labs/pull/53#discussion_r450966294

**Comment by woozyking** [comment]
carousel is probably another good candidate to be graduated out of `react-labs` as an isolated package (like `lumen-table`)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,269 +1,328 @@
...
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 All non-core API/package changes (i.e. changes that do not affect the package delivery to the end users) will be noted under the **Non-Core** category.
 
+## Unreleased
+
+### Changed
+
+- `CSS` - Remove component-level CSS imports.
```
Comment (woozyking) [inline CHANGELOG.md:14]: you can use now v3 of the `release CLI` to generate this portion.

```shell
% release changelog <...params/options>
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,269 +1,328 @@
...
-* radio - Add test-id. - Tests/Radio - Add basic rendering test. (fe14f2e by Sam Sverko)
-* radio - Create component. - Deprecate StyledRadio. - Update stories. - Export FormControl, FormLabel, and RadioGroup from MUI. (a97d15a by Sam Sverko)
-* checkbox - Begin build with custom styles. (3e1edac by Sam Sverko)
-* components - Move deprecated to separate folders. (1db0988 by Sam Sverko)
-* changelog - update (8e750fb by Tamires Lowande)
-* list/list-item - add the option to style timeStatus typography (f021349 by Tamires Lowande)
-* list/list-item - override default style for small items (640b7b6 by Tamires Lowande)
-* devops/package - update tag v1.10.2-alpha (b4484f9 by Tamires)
-* list/list-item - adjust style for elements to work together on small items (a877d62 by Tamires Lowande)
+- chip - update changelog.md (c5061a1 by DoParkEQ)
```
Comment (woozyking) [inline CHANGELOG.md:20]: unnecessary changeset especially that both are rendered as unordered HTML list items, and try to [respect existing conventions](https://github.com/EQWorks/common/blob/master/naming.md#respect-existing-conventions).

even if such change is justified (such as for the sake of personal preference/consistency), it should be derived out as an isolated commit so reviewers can be aware that such commit can be skipped from functional reviews, such as mentioned in [common/git](https://github.com/EQWorks/common/blob/master/git.md#on-commits) (consider this in line with "structural refactor").

Comment (SamSverko) [reply CHANGELOG.md:20]: What's not part of the convention? The use of a `-` instead of a `*`? I'm confused here. The code highlight it also showing changes made by others, not myself...

File: CHANGELOG.md (hunk 1 above)
Comment (SamSverko) [reply CHANGELOG.md]: Does that mean I need to run that command? (when I do, it prints all old updates, not what's unreleased)

or do we only need to run that when pushing a release?

**Review by DoParkEQ — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (woozyking) [reply CHANGELOG.md:14]: yea the change from `*` to `-` can be avoided thus reducing the reviewable changeset.

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,260 +11,312 @@ All non-core API/package changes (i.e. changes that do not affect the package de
...
+- refetchdata - Update `fetchUrl` prop description. (a7fb7d8 by Sam Sverko)
+- refetchdata - Add visuals to loading & error state - Use slow mock http request url for testing. (0ff3266 by Sam Sverko)
+- refetchdata - Move component to `src`. - Export component from `index`. - Update story to reflect import from `index`. (25bca4e by Sam Sverko)
+- refetchdata - Begin building component. (f7f9400 by Sam Sverko)
+- deps - Add axios dep. - Add react-query-devtools to Storybook app wrapper. (5a5ee69 by Sam Sverko)
+- refetchdata - Begin story to build component. (9a6053a by Sam Sverko)
+- deps - Add react-query & devtools deps. (9f3eb25 by Sam Sverko)
+- deps - Upgrade all minor & patch versions [build]. (26e9eff by Sam Sverko)
+- changelog - Reorganize unreleased notes. (03f2e25 by Sam Sverko)
+- switch - Remove prop validations [build]. (d1dfbca by Sam Sverko) \* Update story to include controls for checked and disabled states.
```
Comment (woozyking) [inline CHANGELOG.md:43]: this was formatted with an indented portion for the body/details, like:

* switch - Remove prop validations [build]. (d1dfbca by Sam Sverko)
  * Update story to include controls for checked and disabled states

you may have something dictating and autoformatting from your local editor (such as prettier) that introduced all these unnecessary changes.

**Review by woozyking — APPROVED** [review]
